tensorrt-graphsurgeon/meta.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = "0.4.1" %}
-{% set sha256 = "03ce0427686e19d60919f6ba68dea5446a26ac53f4398bdea4ec96aec30740e7" %}    #[ppc64le]
-{% set trt_tar = "TensorRT-7.0.0.11.CentOS-7.6.ppc64le-gnu.cuda-10.2.cudnn7.6.tar.gz" %} #[ppc64le]
+{% set version = "0.4.5" %}
+{% set sha256 = "32ea84fb87619ec6b90ca3f4bb9d9dd3d5ca12f6a5f2db9777bfc858d9e62464" %}    #[ppc64le]
+{% set trt_tar = "TensorRT-7.2.0.13.CentOS-8.1.ppc64le-gnu.cuda-11.0.cudnn8.0.tar.gz" %} #[ppc64le]
 
-{% set sha256 = "92fab8aa1de9e5bb6904940e840d553bf64caf39ed5fd6a749297c95e3619051" %}    #[x86_64]
-{% set trt_tar = "TensorRT-7.0.0.11.CentOS-7.6.x86_64-gnu.cuda-10.2.cudnn7.6.tar.gz" %}  #[x86_64]
+{% set sha256 = "3d42890424e2a2a155df67aaace45d001a95f12d6781008c5dcf62ea480ea664" %}   #[x86_64]
+{% set trt_tar = "TensorRT-7.2.1.6.CentOS-7.6.x86_64-gnu.cuda-11.0.cudnn8.0.tar.gz" %}  #[x86_64]
 
```

Comment:
Do we not want to support RH7 for ppc64le and RH8 for x86?

Reply:
Or NVIDIA doesn't support it?

Reply:
This is the way they come from Nvidia. You pick the version, then the CUDA level, not the OS level.